README.md

```diff
@@ -13,7 +13,23 @@ Code based on <https://github.com/ebfull/lightning_circuit>.
 
 ## howto
 
-``./get-libsnark && make && ./test``
+### Required packages
+
+* On Ubuntu 16.04 LTS:
+
+        `$ sudo apt-get install build-essential cmake git libgmp3-dev libprocps4-dev python-markdown libboost-all-dev libssl-dev`
+
```

Comment:
I'm not sure about these as the only required packages. Please let me know if you can't build with just these installed.